api/projects/permissions.py

```diff
@@ -31,11 +32,16 @@
 ]
 
 
-class ProjectPermissions(IsAuthenticated):
+class ProjectPermissions(E2ETestPermissionMixin, IsAuthenticated):
     def has_permission(self, request, view):
         """Check if user has permission to list / create project"""
         if not super().has_permission(request, view):
             return False
+        elif (
+            super().has_permission(request, view)
+            and getattr(request, "is_e2e", False) is True
+        ):
+            return True
```

Comment:
@novakzaballa even if this was necessary, the call to `super()` is quite clearly unnecessary as we're early returning on the line above if it's not valid. 

In general though, this is exactly the job that the `E2ETestPermissionMixin` is trying to achieve. Can you have another look at this please?

Reply:
You’re right, the call to super() was no longer necessary and has been corrected.

There was an issue with this part of the [code](https://github.com/Flagsmith/flagsmith/blob/6ef7a742f0f56aef2335da380770dc7f307d53c5/api/projects/permissions.py#L56) that was validating the number of projects and returning `False`, I moved part of the logic exclusively to this validation.

Reply:
In this case, I don't think the mixin is actually doing anything? I would rather see the mixin used, or removed if we can't get it working.

Reply:
I removed it since it doesn't do what we expect.